index.html

```diff
@@ -11,129 +11,189 @@
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="img/twitter.svg" alt="twitter icon">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="img/m.svg" alt="m icon">
+          </a>
+        </li>
+      </ul>
+    </div>
   </section>
+
+<!-- Work section -->
+
   <section class="section-work gen-padding">
     <div class="work-container grid gen-margin">
-      <h2 class="work-header-box primary-font">My Recent Works</h2>
+      <div class="work-header-box">
+        <h2 class="work-header primary-font">My Recent Works</h2>
+        <div class="line"></div>
+      </div>
       <div class="multi-post-card flex">
-        <img src="img/multi.png" alt="Multi-Post illistration" class="multi-post-header-img">
-        <h3 class="multi-post-primary primary-font">Multi-Post Stories</h3>
-        <p class="multi-post-description">
-          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="tags-box flex">
-          <li><a href="#">CSS</a></li>
-          <li><a href="#">HTML</a></li>
-          <li><a href="#">Bootstrap</a></li>
-          <li><a href="#">Ruby</a></li>
-        </ul>
-        <a href="#" class="action-btn">See project</a>
+        <div class="card-img-box">
+          <img src="img/multi.png" alt="Multi-Post illistration" class="multi-post-header-img">
+        </div>
+        <div class="card-text-box flex">
+          <h3 class="multi-post-primary primary-font">Multi-Post Stories</h3>
+          <p class="multi-post-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tags-box flex">
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+          <a href="#" class="action-btn">See project</a>
+        </div>
       </div>
-      <div class="prof-art-card flex">
-        <h3 class="prof-art-primary primary-font">Profesional Art Printing Data</h3>
-        <p class="prof-art-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+      <div class="card card__1 flex">
+        <div class="card-mobile-content">
+          <h3 class="card-primary primary-font">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        </div>
+        <div class="card-desktop-content">
+          
+        </div>
```

Comment:
- [ ] Please I highly recommend sticking to the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=DUj2Dpiew61pAJwT-0). 😄 

- [ ] Please this should be the required design if the `card` is not hovered by the mouse.

![Screenshot (282)](https://user-images.githubusercontent.com/78104544/236359629-f20b6405-d20b-45ac-95a9-e00a8e6f230d.png)

- [ ] Please this should be the design when a user hovers over the `card` item. When the project is hovered on with the mouse the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository. Please note that this change should be implemented on each `card` in the `section-work` class.😄 

![Screenshot (281)](https://user-images.githubusercontent.com/78104544/236359934-39792d63-07cf-4dd9-93f7-9117489c4ccb.png)